enhancedTable/index.ts

```diff
@@ -113,7 +113,8 @@ class TableBuilder {
                         suppressSorting: false,
                         suppressFilter: column.searchDisabled,
                         sort: column.sort,
-                        hide: this.configManager.filteredAttributes.length === 0 ? false : column.column ? !column.column.visible : undefined
+                        notVisible: this.configManager.filteredAttributes.length === 0 ? false : column.column ? !column.column.visible : undefined,
```

Comment:
Since this isn't defined for the ag-grid column definition, it throws a warning in the console. Is there another place to store this value?

Reply:
donethanks